### DISCUSSION TITLE
Table sort with SQL query source

### DISCUSSION THREAD

**Discussion by NicolasFabre81** [Budibase/budibase#10102] [Help] [answered]
Hello, in my SQL query I can make a multi column sort thanks to the ORDER BY sql clause.
But at the data provider level, I'm onliged to sort on one -and only one- column.
So I loose the multi-column sort made in the sql query...
Any workaround ?

Anyway it would be nice to be able to sort a DP on several columns...

**Comment by melohagan** [comment, marked as answer]
Hey @NicolasFabre81 

There's an open issue here: https://github.com/Budibase/budibase/issues/9072
